lib/tokensecurity.js

```diff
@@ -546,6 +562,19 @@ module.exports = function (app, config) {
     return false
   }
 
+  strategy.isDummy = () => {
+    return false
+  }
```

Comment:
If your function is a oneline that returns a value you do it like `() => false`, without the curly braces to save a few lines